site/content/contribute/server/feature-flags.md

```diff
@@ -72,3 +72,20 @@ There are no hard rules on when a feature flag should be used. It is left up to
 ## Examples of feature flags
 
 < add some examples when we create them >
+
+## FAQ
+
+1. What is the expected default value for boolean feature flags? Is it ``true`` or ``false``?
+ - Definitely ``false``. The idea is to use them to slowly roll out a feature. When the code is deployed, the feature flag is not enabled yet. See more details on feature flag rollout timelines here: https://developers.mattermost.com/contribute/server/feature-flags/#timelines-for-rollouts.
+
+2. Is it possible to use a plugin feature flag such as ``PluginIncidentManagement`` to "prepackage" a plugin only on Cloud by only setting a plugin version to that flag on Cloud? Can Self-Managed customers manually set that flag to install the said plugin?
+ - Yes. If you leave the default "" then nothing will happen for Self-Managed installations. You can ask the Cloud team to set split.io/environment to a specific version. Self-Managed installations can set environment variables to set feature flag values.
+
```

Comment:
Can plugins use feature flags to enable minor features aside of the version forcing feature flag?
Yes. You can create feature flags as if they were for the main product, and they will get to the plugin through the Mattermost Config.

(Probably could use better wording)